### PR TITLE
add1

### PR DESCRIPTION
**Pull request by mitnues**

### CODE REVIEW

**Review by raulriato — CHANGES_REQUESTED** [review]
You're disabling `lint` in the project. This is critical. Instead, try fixing the code to work with the lint.

Also, you need to add the `DEMO LINK` to the pull request description

**Review by IanLuan — CHANGES_REQUESTED** [review]
Your DEMO LINK isn’t working. Please run the deploy again. Also, some tests are failing, it looks like the error message isn’t being shown when trying to add a TODO with an empty title. Run `npm run test` and make sure all tests are passing.

**Review by pedro-ruas — CHANGES_REQUESTED** [review]
See the last comments from other mentors.
Links are not working and tests are failing

**Review by pedro-ruas — CHANGES_REQUESTED** [review]
Why is your page requesting a login? That is not in the scope of this project.
Your app has a login, local cache storage for the user, fake items, and reload on add items.
None of this is intended to happen.
This project should use YOUR fixed user ID, and the Mate internal API to handle the todos. There are instructions on how to get your ID and the API URL too.

**Review by raulriato — COMMENTED** [review]
See the last comments from @pedro-ruas. Your project should not have a login page

**Review by raulriato — CHANGES_REQUESTED** [review]
fix it before re-requesting review

**Comment by mitnues** [comment]
sigo tentando. obrigado pela paciência.

**Review by IanLuan — CHANGES_REQUESTED** [review]
You’re almost there. This part of the API is just a bit mixed up. You should follow the same approach used in the react_todo_app_loading_todos task. Right now, you’re using JSONPlaceholder, but you need to use the Mate Academy Students API:
https://mate.academy/students-api.

This is very similar to what you did in react_todo_app_loading_todos. Ideally, reuse that code and only add the add and delete logic. You don’t need the login step. Just get your user ID and hardcode it in the project by following this section:
https://github.com/mate-academy/react_todo-app-loading-todos?tab=readme-ov-file#load-todos-by-userid

Use the provided link, enter your Mate email, get your ID, and use it in the API calls.

If you have any questions, ask in the help chat. The mentors can assist you.

File: src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,267 @@
...
+  const [processingIds, setProcessingIds] = useState<number[]>([]);
+  const [isAddingTodo, setIsAddingTodo] = useState(false);
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  // No user to load
+
+  // Load todos on mount
+  useEffect(() => {
+    setIsLoading(false);
+    setTodos([]); // Start with an empty list
```
Comment (pedro-ruas) [inline src/App.tsx]: This is wrong, you should fetch the todo list from the API

File: src/App.tsx [hunk 2]
```diff
@@ -1,26 +1,267 @@
...
+
+    const newTodoData = {
+      title: trimmedTitle,
+      completed: false,
+    };
+
+    // Create temp todo
+    setTempTodo({
+      id: 0,
+      userId: 3825,
```
Comment (pedro-ruas) [inline src/App.tsx]: Is this number your ID?
Save it into a variable `USER_ID` in the API folder and import here. You will need in both places.
This variable comes in the project, I believe you deleted it

**Review by pedro-ruas — CHANGES_REQUESTED** [review]

**Review by DanielCamposDevX — CHANGES_REQUESTED** [review]
As previous menthors have said, the login is out of the project scope, i recommend you to start the project over. Always keep an eye on the checklist, in the way that you can maintain proximity with the project objective

**Comment by mitnues** [comment]
its doing a dictely login and the last review from the another mentor are fixed. whats wrong now? i cant understand...

**Review by IanLuan — APPROVED** [review]
